docs/src/components/home/sections/AmplifySection.tsx

```diff
@@ -0,0 +1,77 @@
+import { useRef } from 'react';
+import classNames from 'classnames';
+import { Heading, Link, View, Flex } from '@aws-amplify/ui-react';
+import { AmplifyIcon, ServerIcon, LibraryIcon } from '@/components/Icons';
+import { useIntersectionObserver } from '@/components/useIntersection';
+import { HomeFeatureCard } from '../HomeFeatureCard';
+
+export const AmplifySection = ({ platform }) => {
+  const ref = useRef<HTMLDivElement | null>(null);
+  const entry = useIntersectionObserver(ref, {
+    threshold: 0.125,
+    freezeOnceVisible: true,
+  });
+  const isVisible = !!entry?.isIntersecting;
+
+  return (
+    <View
+      ref={ref}
+      as="section"
+      testId="docs-home-section"
+      className={classNames(
+        'docs-home-section',
+        'fade-in',
+        isVisible && 'shown'
+      )}
+    >
+      <Flex direction="column" className="docs-home-subsection--thin">
+        <Heading level={2}>
+          Better <strong>together</strong> with AWS Amplify
+        </Heading>
+
+        <HomeFeatureCard
+          title="Build your application visually in Amplify Studio"
+          icon={AmplifyIcon}
+        >
+          Deploy and manage your app backend with a GraphQL API, User login,
+          File storage and more in minutes. Start small and build to the full
```

Comment:
I thought Amplify Studio was for building components visually and binding them to data? Does it also allow customers to "Deploy and manage your app backend with a GraphQL API, User login, File storage"?

Reply:
Yes, Amplify Console was renamed to Amplify Studio which includes the UI builder functionality as well as the functionality to create backend resources (auth, data, storage, etc.) and manage content: 

<img width="1440" alt="CleanShot 2022-06-21 at 22 23 56@2x" src="https://user-images.githubusercontent.com/321279/174950071-bc34d3b7-0a8b-4c9d-a56c-c7457b63eb3b.png">